packages/artifact/src/internal/download/download-artifact.ts

```diff
@@ -128,10 +128,7 @@ export async function streamExtractExternal(
               if (!createdDirectories.has(path.dirname(fullPath))) {
                 createdDirectories.add(path.dirname(fullPath))
                 await resolveOrCreateDirectory(path.dirname(fullPath)).then(
-                  () => {
-                    entry.autodrain()
```

Comment:
is the `entry.autodrain()` no longer necessary?

Reply:
Yep! Since below we're using `entry.pipe(...)`